### PR TITLE
FEAT: Added homepage.

### PR DESCRIPTION
**Pull request by zoyatanweer**
Files to check : 
- components/Header/Header.jsx
- components/Homepage/Homepage.jsx
- components/Footer/Footer.jsx

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <footer className="footer">
+      <div className="footer-header">
+        <h5 className="footer-title typography-h5">
+          Made with ❤️ by Zoya Tanweer
+        </h5>
+        <h6 className="footer-contact typography-h6">
+          Contact me
+          <ul>
+            <li className="list-item footer-item list-non-bullet">
+              <a href="https://github.com/zoyatanweer">
```
Comment (itsmebipul00) [inline src/components/Footer/Footer.jsx:23]: Suggestion: Link tag to be used

File: src/components/Homepage/Homepage.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  align-items: center;
+  /* padding: 0.5rem; */
+  margin: 1rem 2.5rem 1rem;
+}
+.grid-left {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  /* align-items: center; */
```
Comment (itsmebipul00) [inline src/components/Homepage/Homepage.css]: remove all comments

**Review by itsmebipul00 — APPROVED** [review]
Clean code. Good to merge

**Review by dev-prabhat — APPROVED** [review]
Code is fine and readable, 
Good to  merge

File: src/Assets/index.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import homepage from "./Images/homepage.jpg";
```
Comment (sumitk-singh) [inline src/Assets/index.js]: Suggestion: For better clarity **homepage** can be as **homePageImg** as it is image.

File: src/components/Footer/Footer.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+}
+.footer-item:hover {
+  color: var(--primary-color);
+}
+ul li.footer-item {
+  color: var(--background-color);
+}
+.footer-copyright {
+  display: flex;
+  flex-direction: row;
```
Comment (sumitk-singh) [inline src/components/Footer/Footer.css]: The default value of flex-direction is **row**, you may remove it.

File: src/index.css [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+  margin: 5px 15px 5px 0;
+  padding: 5px;
+  width: 20rem;
+  line-height: 1;
+}
+
+/* action-btn */
+.desktop-action-btn {
+  display: flex;
+  flex-direction: row;
```
Comment (sumitk-singh) [inline src/index.css]: **row** is the default value

**Review by sumitk-singh — APPROVED** [review]
Clean code and UI, Good to merge.

File: src/App.js [hunk 6]
```diff
@@ -1,8 +1,17 @@
-import { Routes, Route } from "react-router-dom";
+import { Header } from "./components/Header/Header";
+import { Homepage } from "./components/Homepage/Homepage";
+import { Footer } from "./components/Footer/Footer";
```
Comment (zoef-yt) [inline src/App.js]: use index.js

File: src/components/Footer/Footer.css [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+.footer {
+  background-color: var(--footer-bg-color);
+  color: var(--background-color);
+  font-size: 2rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 16rem;
+  padding: 12rem;
+  text-align: center;
```
Comment (zoef-yt) [inline src/components/Footer/Footer.css]: You don't need text-align:center;

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (zoef-yt) [reply src/components/Footer/Footer.jsx:23]: Using the Link tag will not take you to the external site and this is an external site.

File: src/components/Header/Header.jsx [hunk 8]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link, NavLink } from "react-router-dom";
+
+import "../Header/Header.css";
+
+const Header = () => {
+  return (
+    <>
```
Comment (zoef-yt) [inline src/components/Header/Header.jsx]: you don't need fragment here

File: src/components/Homepage/Homepage.css [hunk 9]
```diff
@@ -0,0 +1,44 @@
+.homepage {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  /* padding: 0.5rem; */
```
Comment (zoef-yt) [inline src/components/Homepage/Homepage.css]: remove commented code.

File: src/components/Homepage/Homepage.css [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+  background-color: var(--primary-color);
+  color: var(--background-color);
+  font-size: 1.7rem;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.btn-primary:hover {
+  /* background-color: var(--secondary-color); */
```
Comment (zoef-yt) [inline src/components/Homepage/Homepage.css]: remove commented code.

File: src/components/Homepage/Homepage.jsx [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+      <div class="grid-left mg-sm">
+        <h1 className="heading-h1">Lorem ipsum dolor sit amet</h1>
+        <p className="para-medium">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam.
+        </p>
+        <button class="btn box-shadow btn-primary">Create Note</button>
+      </div>
+      <img className="homepage-cover" src={homeImage} />
```
Comment (zoef-yt) [inline src/components/Homepage/Homepage.jsx]: add an alt tag

File: src/index.css [hunk 12]
```diff
@@ -0,0 +1,215 @@
...
+  font-weight: 500;
+  padding: 1.5rem;
+}
+.list-item:hover {
+  font-weight: 600;
+  color: var(--primary-color);
+}
+.navigation-container {
+  background-color: var(--background-color);
+  /* box-shadow: 10px 10px 10px -12px var(--navigation-container-shadow); */
```
Comment (zoef-yt) [inline src/index.css]: remove commented code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Comment by zoyatanweer** [comment]
Thankyou guys for all your reviews.

**Pull request merged**